apps/src/code-studio/components/progress/UnitCalendarLessonChunk.jsx

```diff
@@ -0,0 +1,154 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {unitCalendarLessonChunk} from '@cdo/apps/templates/progress/unitCalendarLessonShapes';
+import color from '@cdo/apps/util/color';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+
+const styles = {
+  box: {
+    margin: 5,
+    color: '#333',
+    textDecorationLine: 'none'
+  },
+  boxContent: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    textAlign: 'center',
+    fontFamily: '"Gotham 4r", sans-serif',
+    height: '100%'
+  },
+  assessment: {
+    border: '2px solid ' + color.purple
+  },
+  assessmentHover: {
+    border: '2px solid ' + color.purple,
+    backgroundColor: color.purple,
+    color: 'white'
+  },
+  instructional: {
+    border: '2px solid ' + color.teal
+  },
+  instructionalHover: {
+    border: '2px solid ' + color.teal,
+    backgroundColor: color.teal,
+    color: color.white
+  },
+  isNotStart: {
+    borderLeftStyle: 'dashed'
+  },
+  isNotEnd: {
+    borderRightStyle: 'dashed'
+  },
+  titleText: {
+    width: '100%',
+    paddingRight: 15
+  },
+  iconSection: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    height: '100%',
+    paddingLeft: 2,
+    paddingTop: 2,
+    paddingBottom: 2,
+    boxSizing: 'border-box'
+  }
+};
+
+class UnitCalendarLessonChunk extends Component {
+  static propTypes = {
+    minuteWidth: PropTypes.number.isRequired,
+    lesson: unitCalendarLessonChunk,
```

Comment:
nit: I think this prop should be called lessonChunk to call out that its possibly just one part of the whole lesson on the calendar.